services/keybase/keybase-profile.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const { BaseJsonService } = require('..')
+
+module.exports = class KeybaseProfile extends BaseJsonService {
+  static get apiVersion() {
+    throw new Error(`apiVersion() is not implemented for ${this.name}`)
+  }
+
+  async fetch({ schema, options }) {
+    const apiVersion = this.constructor.apiVersion
+    const url = `https://keybase.io/_/api/${apiVersion}/user/lookup.json`
```

Comment:
Can you add an inline comment here that provides a link to the API documentation?

Reply:
Done :+1: